web/src/components/Button.tsx

```diff
@@ -6,7 +6,7 @@ import { makeStyles } from '@material-ui/core/styles'
 interface CustomProps {
   loading?: boolean
   hasSpinner?: boolean
-  spinnerColor?: string
+  spinnercolor?: string
```

Comment:
We can increase test coverage + make this component simpler by just removing `hasSpinner` and making `spinnercolor` non optional, or just removing it and using the theme color. Also make the `loading` non optional.

There are currently no instances of this component where it doesn't have a spinner / loading props -- it doesn't need to be this general until we have a use case for it.

Reply:
The epiphany I had just now while making my coffee is that I can revert this file change in this PR, because I already made the change in my unblocked PR. Should help with increasing test coverage on this PR.